### ISSUE TITLE
Missing support for mocha '--reporter-options' parameter

### ISSUE THREAD

**Issue by bencao** [Rowno/grunt-mocha-cli#32]
Mocha already shipped a xunit reporter which can write a xml string into a output file.
In order to tell mocha which output file it is, we need to pass on the '--reporter-options' parameter which is missing in current grunt-mocha-cli.

The parameter is a string base on mocha command help:

```
  option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options')
```

Could you please add support for that string parameter?

**Comment by bencao** [comment]
PR created for this issue https://github.com/Rowno/grunt-mocha-cli/pull/33

**Comment by Rowno** [comment]
Released in version 1.14.0. :sparkles:
